spec/spec_helper.rb

```diff
@@ -1,5 +1,8 @@
 # frozen_string_literal: true
 
+require 'fileutils'
+require 'tmpdir'
```

Comment:
I think these requires should be in `spec/support/file_helper.rb` as that is the file that uses them.